src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbolBase.cs

```diff
@@ -59,7 +60,7 @@ internal abstract class SourceOrdinaryMethodSymbolBase : SourceMemberMethodSymbo
 
             var isMetadataVirtualIgnoringModifiers = methodKind == MethodKind.ExplicitInterfaceImplementation; //explicit impls must be marked metadata virtual
 
-            this.MakeFlags(methodKind, declarationModifiers, returnsVoid, isExtensionMethod, isMetadataVirtualIgnoringModifiers);
+            this.MakeFlags(methodKind, declarationModifiers, returnsVoid, isExtensionMethod: isExtensionMethod, isNullableEnabled: isNullableEnabled, isMetadataVirtualIgnoringModifiers: isMetadataVirtualIgnoringModifiers);
```

Comment:
>isExtensionMethod [](start = 74, length = 17)

Nit: all of these named parameters can remove their names.

Reply:
There are several consecutive `bool` parameters so I think names are useful to avoid bugs if parameters are reordered.

---
In reply to: [546875151](https://github.com/dotnet/roslyn/pull/49953#discussion_r546875151) [](ancestors = 546875151)